clients/admin-ui/src/features/common/table/v2/TableActionBar.tsx

```diff
@@ -8,6 +8,7 @@ export const TableActionBar = ({ children, ...props }: StackProps) => (
     borderWidth="1px"
     borderBottomWidth="0px"
     borderColor="gray.200"
+    zIndex={11}
```

Comment:
The THead on FidesTable has a z-index of 10, so this needed to be higher in order for the popover to render correctly.